pom.xml

```diff
@@ -333,6 +333,12 @@
                 <artifactId>commons-lang</artifactId>
                 <version>2.6</version>
             </dependency>
+            <dependency>
+                <groupId>com.sun.activation</groupId>
```

Comment:
Move this up above the `com.tdunning` reference.

Reply:
I have removed this and used the existing `jakarta.activation-api`